doc/lua_api.md

```diff
@@ -4111,9 +4111,10 @@ On some specific cases, server translation could be useful. For example, filter
 a list on labels and send results to client. A method is supplied to achieve
 that:
 
-`minetest.get_translated_string(lang_code, string)`: Translates `string` using
-translations for `lang_code` language. It gives the same result as if the string
-was translated by the client.
+`minetest.get_translated_string(lang_code, S(string))`: resolves translations in
```

Comment:
sorry, didn't see that before but: why is the `S` here?
the text already explains that the input is supposed to contain translations markup
and it is in fact correct to do e.g. `minetest.get_translated_string("", "foo\n" .. def.description .. "\nbar")`. No `S` in sight.

Reply:
I think people can still misunderstand such explanation. I've provided an example instead of listing the two translation methods, let me know